prime-router/docs/getting-started/README.md

```diff
@@ -14,6 +14,7 @@
 - [Debugging](#debugging)
 - [Running the frontend](#running-the-frontend)
 - [Running the static site](#running-the-static-site)
+- [Push a sample message through the app](#push-a-sample-message-through-the-app)
```

Comment:
```suggestion
- [Submit a sample report to ReportStream](#push-a-sample-message-through-the-app)
```

Reply:
This language better matches our preferred. Use this language elsewhere, like in the heading below.